src/base1/cockpit.js

```diff
@@ -1082,14 +1089,13 @@ function factory() {
     };
 
     /* Not public API ... yet? */
-    cockpit.hint = function hint(name, host) {
-        if (!host)
-            host = default_host;
-
-        var options = { "command": "hint",
-                        "hint": name,
-                        "host": host };
-
+    cockpit.hint = function hint(name, options) {
```

Comment:
If we always send the "ignore_health_check" hint on towards the parent, we can not switch off the health check when a component uses a WebSocket directly, such as when it is not embedded into the shell.